tape/src/main/java/com/squareup/tape/CorruptionException.java

```diff
@@ -0,0 +1,12 @@
+package com.squareup.tape;
+
+import java.io.IOException;
+
+/**
+ * Exception class representing corruption in the Tape queue.
+ */
+public class CorruptionException extends IOException {
```

Comment:
Why do we need a custom subclass in the public API?

Also: make final

Reply:
We don't strictly need it. I thought it would provide some convenience at very little cost other than semver version bump. Perhaps I am missing something; just preferring to minimize the public API changes at all costs?

Reply:
Removed